docs: work with remote pinning services in webui (#821)
ipfs/ipfs-docs: docs/how-to/work-with-pinning-services.md

description: Learn how to use or create remote pinning services with IPFS, the InterPlanetary File System.
---

# Work with remote pinning services

> [[toc]]

Reviewer: Can we please avoid these? It's duplication for people who are viewing this on docs.ipfs.io.

Author: In this specific case toc here is useful, because it provides links to GUI and CLI sections, while the outline in the left menu does not:

>  ![2021-06-25--00-02-32](https://user-images.githubusercontent.com/157609/123338193-aeaf0780-d548-11eb-945e-8e697063785e.png)

Makes it way easier for people to jump to specific guide.

::: tip

> If you're interested in creating your own pinning service for your own personal or shared use, you can [generate client and server from the OpenAPI spec](https://github.com/ipfs/pinning-services-api-spec#code-generation), reducing the development time, or [reuse one of existing solutions](https://github.com/ipfs/pinning-services-api-spec#adoption)

Reviewer: ```suggestion
If you're interested in creating your own pinning service for your own personal or shared use, you can [generate client and server from the OpenAPI spec](https://github.com/ipfs/pinning-services-api-spec#code-generation), reducing the development time, or [reuse an existing solution](https://github.com/ipfs/pinning-services-api-spec#adoption)
```
